roles/openldap/tasks/install-configure.yml

```diff
@@ -0,0 +1,80 @@
+- name: install the openldap packages
+  apt: name={{ item }} state=installed
+  with_items: "{{ openldap.ldap_pkgs }}"
+
+- name: create directory for ldif files
+  file: path={{ openldap.config_dir }}
+        state=directory
+        owner=root
+        group=root
+        mode=0400
+
+- name: copy templates
+  template: src={{ item }}
+            dest={{ openldap.config_dir }}/{{ item | basename }}
```

Comment:
The files are used the ldap commands and not really needed for openldap functioning. I had initially listed the ldif files individually as with_items. I was asked to move the file names to configuration in previous review comments in  zrs233/ursula, this was not a sensible thing to do as the template files are not some kind of overrideable config, the content is templated anyway.

To avoid a long list in the template module call, I decided to use with_fileglob, that started resolving the dest differently and had to use basename, will try to refactor the source dirs to not use basename.